asammdf/gui/widgets/file.py

```diff
@@ -639,15 +647,18 @@ def set_raster_type(self, event):
             self.raster_channel.setCurrentIndex(0)
             self.raster.setEnabled(True)
 
+    def update_all_channel_trees(self):
+        widgetList = [self.channels_tree, self.filter_tree]
+        for widget in widgetList:
+            self._update_channel_tree(widget=widget)
+
     def _update_channel_tree(self, index=None, widget=None):
 
         if widget is None:
             widget = self.channels_tree
         if widget is self.channels_tree and self.channel_view.currentIndex() == -1:
             return
-        elif widget is self.filter_tree and (
-            self.filter_view.currentIndex() == -1 or not self._show_filter_tree
```

Comment:
the idea here was to have lazy loading of the filter channel list since it is not used as much